tss-esapi-sys/build.rs

```diff
@@ -59,19 +69,158 @@ pub mod tpm2_tss {
     /// The installed tpm2-tss libraries that are of
     /// interest.
     pub struct Installation {
-        _tss2_sys: Library,
+        tss2_sys: Library,
         tss2_esys: Library,
         tss2_tctildr: Library,
         tss2_mu: Library,
         tss2_tcti_tbs: Option<Library>,
     }
 
     impl Installation {
+        fn platform_args() -> Option<Vec<String>> {
+            cfg_if::cfg_if! {
+                if #[cfg(windows)] {
+                    let mut clang_args: Vec<String> = Vec::new();
+                    let hklm = winreg::RegKey::predef(winreg::enums::HKEY_LOCAL_MACHINE);
+                    let sdk_entry = hklm.open_subkey("SOFTWARE\\WOW6432Node\\Microsoft\\Microsoft SDKs\\Windows\\v10.0").unwrap();
+                    let installation_path: String = sdk_entry.get_value("InstallationFolder").unwrap();
+                    let ip_pb = PathBuf::from(installation_path).join("Include");
+                    let windows_sdk = ip_pb.join("10.0.17134.0");
+                    clang_args.push(format!("-I{}", windows_sdk.join("ucrt").display()));
+                    clang_args.push(format!("-I{}", windows_sdk.join("um").display()));
+                    clang_args.push(format!("-I{}", windows_sdk.join("shared").display()));
+                    return Some(clang_args);
+                }
+                else {
+                    return None;
```

Comment:
Remove the return's should complain about `return Some` if you tried to compile this on windows as well.